### PR TITLE
Re-aligned with Flexbox and css media query

### PR DESCRIPTION
**Pull request by Mattimus333**
Removed the nbsp's from the label, aligned the labels/inputs with some flexbox.  It expands the height of search-form div and reorders the items so they stack under eachother when the viewport is 500px or smaller.  This allows the inputs to remain fairly long.  

Could possibly bump up the max width of the media query.

### CODE REVIEW

File: src/components/PublicPursuances/PublicPursuanceList.js [hunk 1]
```diff
@@ -88,18 +88,19 @@ class PublicPursuanceList extends Component {
       <div className="pursuance-list">
         <h2 className="dash-box-title">Search All Public {PROJECTS_CAPITAL}</h2>
         <div className="public-pursuance-search-form">
-          <label>Search by tag:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</label>
+          <label htmlFor="a">Search by tag:</label>
           <input
+            id="a"
```
Comment (elimisteve) [inline src/components/PublicPursuances/PublicPursuanceList.js]: @Mattimus333 More descriptive `id`s, please! `search-by-tag` or something.

File: src/components/PublicPursuances/PublicPursuanceList.js [hunk 2]
```diff
@@ -88,18 +88,19 @@ class PublicPursuanceList extends Component {
...
             type="text"
-            placeholder="PrisonReform"
+            placeholder="Prison Reform"
```
Comment (elimisteve) [inline src/components/PublicPursuances/PublicPursuanceList.js]: Switch back, please

File: src/components/PublicPursuances/PublicPursuanceList.js [hunk 3]
```diff
@@ -88,18 +88,19 @@ class PublicPursuanceList extends Component {
...
             onChange={this.onChangeTag}
           />
-          <br />
-          <label>Search {PROJECT} descriptions:</label>
+          <label htmlFor="b">Search {PROJECT} descriptions:</label>
           <input
+            id="b"
             type="text"
-            placeholder="prison-industrial complex"
+            placeholder="Prison Industrial Complex"
```
Comment (elimisteve) [inline src/components/PublicPursuances/PublicPursuanceList.js]: Switch back, too :-)

File: src/components/PublicPursuances/PublicPursuances.css [hunk 4]
```diff
@@ -1,7 +1,14 @@
 .public-pursuance-search-form {
+  display: flex;
+  align-content: flex-start;
+  flex-flow: column wrap;
+  height: 70px;
   margin-top: 5px;
 }
 
+.public-pursuance-search-form label[for='a'] { order: -2; }
```
Comment (elimisteve) [inline src/components/PublicPursuances/PublicPursuances.css]: @Mattimus333 Let's avoid CSS one-liners do it more like

```css
.public-pursuance-search-form label[for='search-by-tag'] {
    order: -2;
}
```

File: src/components/PublicPursuances/PublicPursuances.css [hunk 5]
```diff
@@ -1,7 +1,14 @@
...
+.public-pursuance-search-form label[for='b'] { order: -1; }
```
Comment (elimisteve) [inline src/components/PublicPursuances/PublicPursuances.css]: Same here (break into multiple lines)

File: src/components/PublicPursuances/PublicPursuances.css [hunk 6]
```diff
@@ -28,3 +36,11 @@
 .pursuance-description h3 {
   color: #50b3fe;
 }
+
+@media (max-width:500px) {
+  .public-pursuance-search-form {
+    height: 140px;
+  }
+
+  .public-pursuance-search-form label[for='b'] { order: 0; }
```
Comment (elimisteve) [inline src/components/PublicPursuances/PublicPursuances.css]: Multi-line, please

File: src/components/PublicPursuances/PublicPursuanceList.js (hunk 3 above)
Comment (Mattimus333) [reply src/components/PublicPursuances/PublicPursuanceList.js]: haha I got a little carried away with these.

**Comment by Mattimus333** [comment]
Cleaned things up, should be good to go.

**Comment by elimisteve** [comment]
@Mattimus333 Merged! :tada: Nice work :-D .

**Pull request merged**